addon/index.js

```diff
@@ -118,3 +118,15 @@ export function selectOptionInRadioGroup(label, option, options) {
   const element = withinTL(parent).getByRole('radio', { name: option });
   return click(element);
 }
+
+function _cleanHtml(html) {
+  return html.replace('<!---->', '').trim();
+}
+
+export function getByTextWithHtml(textToFind) {
```

Comment:
Juste une petite suggestion : de la même manière que la fonction `_cleanHtml` a un argument `html`, est-ce que ça ne serait pas encore un peu plus clair si le paramètre de cette fonction était nommé `html` plutôt que `textToFind` ?
```suggestion
export function getByTextWithHtml(html) {
```